src/edu/csus/ecs/pc2/clics/API202306/CLICSProblemScore.java

```diff
@@ -0,0 +1,76 @@
+// Copyright (C) 1989-2024 PC2 Development Team: John Clevenger, Douglas Lane, Samir Ashoo, and Troy Boudreau.
+package edu.csus.ecs.pc2.clics.API202306;
+
+import java.util.HashMap;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import edu.csus.ecs.pc2.core.StringUtilities;
+import edu.csus.ecs.pc2.core.Utilities;
+import edu.csus.ecs.pc2.core.model.ContestTime;
+import edu.csus.ecs.pc2.core.standings.ProblemSummaryInfo;
+
+/**
+ * CLICS Score (for a problem)
+ * Contains information about the score a team received for a single problem
+ * 
+ * @author John Buck
+ *
+ */
+
+public class CLICSProblemScore {
+
+    @JsonProperty
+    private String problem_id;
+
+    @JsonProperty
+    private int num_judged;
+
+    @JsonProperty
+    private int num_pending;
+
+    @JsonProperty
+    private boolean solved;
+
+// Not needed for pass-fail contest
+//  @JsonProperty
+//  private double score;
+
+    @JsonProperty
+    private String time;
+
+    /**
+     * Fill in API problem score information properties (for scoreboard endpoint)
+     * 
+     * @param probEleToShort hashmap for mapping problem elementid to shortname
+     * @param versionInfo
+     */
+    public CLICSProblemScore(HashMap<String, String> probEleToShort, ProblemSummaryInfo psi) {
+        num_judged = Utilities.nullSafeToInt(psi.getAttempts(), 0);
+        num_pending = Utilities.nullSafeToInt(psi.getIsPending(), 0);
```

Comment:
Same issue with the call to `getIsPending()`; as far as I can tell _no code anywhere actually sets that value_;

Reply:
See above.